examples/chef/common/stubs.cpp

```diff
@@ -14,3 +48,79 @@ bool emberAfPluginDoorLockOnDoorUnlockCommand(chip::EndpointId endpointId, chip:
 {
     return true;
 }
+
+// ************************************
+// Bridge Stubs
+// ************************************
+
+namespace {
+
+class BridgedActionsAttrAccess : public AttributeAccessInterface
+{
+public:
+    // Register for the Bridged Actions cluster on all endpoints.
+    BridgedActionsAttrAccess() : AttributeAccessInterface(Optional<EndpointId>::Missing(), BridgedActions::Id) {}
+
+    CHIP_ERROR Read(const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder) override;
+
+private:
+    static constexpr uint16_t ClusterRevision = 1;
+
+    CHIP_ERROR ReadActionListAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder);
+    CHIP_ERROR ReadEndpointListAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder);
+    CHIP_ERROR ReadSetupUrlAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder);
+    CHIP_ERROR ReadClusterRevision(EndpointId endpoint, AttributeValueEncoder & aEncoder);
+};
+
+constexpr uint16_t BridgedActionsAttrAccess::ClusterRevision;
+
+CHIP_ERROR BridgedActionsAttrAccess::ReadActionListAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder)
+{
+    // Just return an empty list
+    return aEncoder.EncodeEmptyList();
+}
+
+CHIP_ERROR BridgedActionsAttrAccess::ReadEndpointListAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder)
+{
+    // Just return an empty list
+    return aEncoder.EncodeEmptyList();
+}
+
+CHIP_ERROR BridgedActionsAttrAccess::ReadSetupUrlAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder)
+{
+    const char SetupUrl[] = "https://example.com";
+    return aEncoder.Encode(chip::Span<const char>(SetupUrl, strlen(SetupUrl)));
```

Comment:
```suggestion
    return aEncoder.Encode(chip::CharSpan::fromCharString(SetupUrl));
```

Reply:
This replicates the same stubs used in all clusters app. But fair point, the only implementation really needed here is `MatterBridgedActionsPluginServerInitCallback`. I'll make the PR smaller without `BridgedActionsAttrAccess` class, and make another one with that custom implementation if chef users request it